.github/workflows/test.yml

```
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ['1.23', '1.24']
```

Let's only use `[stable, oldstable]` so we don't have to update this with new Go releases 

```suggestion
        go-version: ['oldstable', 'stable']
```

---

Updated.